infra/modules/s3/emails.tf

```diff
@@ -0,0 +1,30 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+resource "aws_s3_bucket_acl" "my_bucket" {
```

Comment:
Instead of 'my_bucket' we can be more specific. I think something like 'email_forwarding_bucket' would be clearer

Reply:
Good point thanks - I'll change it to `email_forwarding_bucket`